doc/api/colors_api.rst

```diff
@@ -14,42 +14,307 @@
    :no-members:
    :no-inherited-members:
 
-Classes
--------
+Color Conversion tools
+----------------------
+
+.. autosummary::
+   :toctree: _as_gen/
+   :template: autosummary.rst
+
+   is_color_like
+   same_color
+
+   hsv_to_rgb
+   rgb_to_hsv
+
+   to_hex
+   to_rgb
+   to_rgba
+   to_rgba_array
+
+   get_named_colors_mapping
+
+
+.. _norms_and_colormaps:
+
+Normalization and Colormapping of Continuous Data
+-------------------------------------------------
+
+Some `~.artist.Artist` classes can map an array of input data to RGBA
```

Comment:
There is a TOC aspect to this file, and then you've added an explanation.  a) not 100% sure the explanation belongs here, versus an intermediate/advanced tutorial.  Most long-form explanations are tutorials currently (i.e. transform stack).  I'm not adamantly advocating for that, but it would be more consistent with current practice.  It would also make your examples easier to write.  

Regardless, this large narrative in the middle of the TOC makes the TOC very hard to parse.  At the very least, this should go at the end?